### PR TITLE
Rn entry 2020.11.0

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?**
Release Notes for 2020.11.0 Monthly
- **Is this PR related to a Monolith release?**
Yes. 2020.11.0

### Resolves:

* [OKTA-343324](https://oktainc.atlassian.net/browse/OKTA-343324)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2020-11-0/index.md [hunk 1]
```diff
@@ -6,4 +6,54 @@ title: Okta API Products Release Notes
...
+| [MyAccount API is now in Early Access (EA)](#myaccount-api-is-now-in-early-access-ea) | November 4, 2020         |
+| [Bug Fixed in 2020.11.0](#bug-fixed-in-2020-11-0) | November 4, 2020        |
+
+### Inclusive language and terminology
+
+Okta is focused on the adoption of inclusive language and communication. Some long-standing industry terminology and expressions have been updated in this release. More updates will be made in future releases.
+
+The descriptive information returned on both the invalid redirect URI and invalid logout URI error pages has been updated to remove the term "whitelisted". <!--OKTA-336922-->
+
+### System Logs API adds additional filter expressions
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-11-0/index.md:25]: Logs -> Log

File: packages/@okta/vuepress-site/docs/release-notes/2020-11-0/index.md [hunk 2]
```diff
@@ -6,4 +6,54 @@ title: Okta API Products Release Notes
...
+
+To help you manage zones in your organization, the Early Access [Zones API](/docs/reference/api/zones/) now includes the `usage` attribute. There are two types of zones: Policy Network Zones and Block List Network Zones. <!--OKTA-333653-->
+
+### Client-based rate limiting is now GA in Production
+
+[Client-based rate limiting](/docs/reference/rl-clientbased/) for the `/authorize` endpoint is now available in production orgs. It provides granular isolation between requests made to the `/authorize` endpoint by using a combination of the Client ID, user's IP address, and Okta device identifier. This isolates rogue OAuth clients and bad actors, ensuring valid users and applications don't run into rate limit violations.
+
+This feature will be available to orgs in Okta Production cells on November 9, 2020. <!--OKTA-342520-->
+
+### User Consent for OAuth 2.0 and OpenID Connect Flows is rolling out to GA in Production
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-11-0/index.md:39]: Flows -> flows

**Review by susanharper-okta — APPROVED** [review]
a couple nits. I'll approve to not hold you up.

File: packages/@okta/vuepress-site/docs/release-notes/2020-11-0/index.md (hunk 1 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2020-11-0/index.md:25]: Fixed

File: packages/@okta/vuepress-site/docs/release-notes/2020-11-0/index.md (hunk 2 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2020-11-0/index.md:39]: Fixed.

**Pull request merged**